code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -295,7 +295,33 @@ GLOBAL_LIST_EMPTY(TabletMessengers) // a list of all active messengers, similar
 	else if(atom_integrity < max_integrity)
 		. += span_warning("It is damaged.")
 
-	. += get_modular_computer_parts_examine(user)
+	var/obj/item/computer_hardware/hard_drive/hdd = all_components[MC_HDD]
+	for(var/datum/computer_file/app_examine as anything in hdd.stored_files)
+		if(app_examine.on_examine(src, user))
+			. += app_examine.on_examine(src, user)
+
+	var/obj/item/computer_hardware/card_slot/card_slot = all_components[MC_CARD]
+	var/obj/item/computer_hardware/card_slot/card_slot2 = all_components[MC_CARD2]
+	var/multiple_slots = istype(card_slot) && istype(card_slot2)
+	if(card_slot)
+		if(card_slot?.stored_card || card_slot2?.stored_card)
+			var/obj/item/card/id/first_ID = card_slot?.stored_card
+			var/obj/item/card/id/second_ID = card_slot2?.stored_card
+			var/multiple_cards = istype(first_ID) && istype(second_ID)
+			if(Adjacent(user))
+				. += "It has [multiple_slots ? "two slots" : "a slot"] for identification cards installed[multiple_cards ? " which contain [first_ID] and [second_ID]" : ", one of which contains [first_ID ? first_ID : second_ID]"]."
+			else
+				. += "It has [multiple_slots ? "two slots" : "a slot"] for identification cards installed, [multiple_cards ? "both of which appear" : "and one of them appears"] to be occupied."
+			. += span_info("Alt-click [src] to eject the identification card[multiple_cards ? "s":""].")
+		else
+			. += "It has [multiple_slots ? "two slots" : "a slot"] installed for identification cards."
+
+	var/obj/item/computer_hardware/printer/printer_slot = all_components[MC_PRINT]
+	if(printer_slot)
+		. += "It has a printer installed."
+		if(Adjacent(user))
+			. += "The printer's paper levels are at: [printer_slot.stored_paper]/[printer_slot.max_paper].</span>"
```

Comment:
I forget, is it normal for none of these to have span_notice? Yes, right?

Reply:
to be honest i didnt touch this, it was just copied from shared computer parts, so i have no clue

Reply:
well if you examine and its not plain text then its right